calico/openstack/mech_calico.py

```diff
@@ -607,6 +621,11 @@ def felix_heartbeat_thread(self, hostname):
                 else:
                     LOG.error("Unexpected response from Felix on %s: %s"
                               % (hostname, rsp))
+            except Again:
+                LOG.error("No response from Felix within allowed time (%sms)" %
+                          HEARTBEAT_RESPONSE_TIMEOUT)
```

Comment:
I think the comment is meaning that the poll call will raise an exception - but I'm happy to clarify that, if that will address your concern here.